atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -215,14 +184,87 @@ def _send_mail(self, receivers_list, subject, body):
 
         s = None
         try:
-            s = smtplib.SMTP(self.smtp_uri)
+            s = smtplib.SMTP(self.smtp_host)
             s.sendmail(self.from_address, receivers_list, msg.as_string())
```

Comment:
...but smtplib.SMTP.sendmail() expects a list there... a bare string is treated as a singleton list.